pt_BR.lang

```diff
@@ -81,237 +83,264 @@ replaymod.gui.exit=Sair do Replay
 
 replaymod.gui.settings=Configurações
 
-replaymod.gui.keyframerepo.delete=Tem certeza de que deseja excluir esses keyframes?
-replaymod.gui.keyframerepo.overwrite=Tem certeza de que deseja substituir esses keyframes?
+replaymod.gui.keyframerepo.delete=Deseja mesmo excluir esses keyframes?
+replaymod.gui.keyframerepo.overwrite=Deseja mesmo substituir esses keyframes?
 
 replaymod.gui.restorereplay1=Parece que o Minecraft não fechou normalmente.
 replaymod.gui.restorereplay2=O replay "%1$s" não foi salvo corretamente.
-replaymod.gui.restorereplay3=Você deseja recuperá-lo?
+replaymod.gui.restorereplay3=Deseja mesmo recuperá-lo?
 
-replaymod.gui.loadentitytracker=Carregando posições da entidade: %%d%%%%
-replaymod.gui.loadquickmode=Carregando o Modo Rápido: %%d%%%%
-replaymod.gui.noquickmode=O Modo Rápido está disponível apenas no Minecraft %1$s e acima
-replaymod.gui.minimalmode.unsupported=Este recurso não está disponível para sua versão atual do Minecraft.
-replaymod.gui.minimalmode.supportedversion=(apenas Minecraft %1$s é suportado pela sua versão do Replay Mod)
+replaymod.gui.loadentitytracker=Carregando as posições da entidade: %%d%%%%
+replaymod.gui.loadquickmode=Carregando o modo rápido: %%d%%%%
+replaymod.gui.noquickmode=O modo rápido está disponível apenas no Minecraft %1$s e superior
+replaymod.gui.minimalmode.unsupported=Este recurso não está disponível na versão atual do Minecraft.
+replaymod.gui.minimalmode.supportedversion=(apenas o Minecraft %1$s é suportado pela sua versão do Replay Mod)
 
 #Only change these if it's neccessary
-replaymod.gui.replayviewer=Visualizar Replay
+replaymod.gui.replayviewer=Replays
 
 #Replay Viewer GUI
-replaymod.gui.viewer.rename.name=Nome do Replay
-replaymod.gui.viewer.replayfolder=Abrir pasta do Replay...
+replaymod.gui.viewer.rename.name=Nome do replay
+replaymod.gui.viewer.replayfolder=Abrir pasta de replays...
+replaymod.gui.viewer.bulkrender=Renderizar %1$d vídeos
 
-replaymod.gui.viewer.delete.linea=Tem certeza de que deseja excluir este Replay?
-replaymod.gui.viewer.delete.failed1=Seu sistema operacional não nos permitiu remover o Replay. There is
-replaymod.gui.viewer.delete.failed2=Você terá que fazer apaga-lo manualmente.
-replaymod.gui.viewer.delete.lineb='%1$s' será perdido para sempre! (muito tempo!)
+replaymod.gui.viewer.delete.linea=Deseja mesmo excluir este replay?
+replaymod.gui.viewer.delete.failed1=Seu sistema operacional não nos permitiu excluir o replay.
+replaymod.gui.viewer.delete.failed2=Não há nada que possamos fazer neste caso. Você deve fazer isto por conta própria.
```

Comment:
```suggestion
replaymod.gui.viewer.delete.failed2=não há nada que possamos fazer neste caso. Você deve fazer isto por conta própria.
```

Reply:
This change isn't necessary since that string is another line and should start with an uppercase.